packages/grafana-data/src/types/data.ts

```diff
@@ -81,6 +81,17 @@ export interface QueryResultMeta {
    */
   pathSeparator?: string;
 
+  /**
+   * TypeVersion is the version of the Type property. Versions greater than 0.0 correspond to the dataplane
+   * contract documentation https://github.com/grafana/grafana-plugin-sdk-go/tree/main/data/contract_docs.
+   */
+  typeVersion?: string;
+
+  /**
+   * FrameType is a 2 number version (Major / Minor)
+   */
+  frameTypeVersion?: string;
```

Comment:
Only adding one Property, `typeVersion` (which indicated the version associated with the value of the `type` property). So probably good to put them next to each other in the code. (In the BE PR, `FrameMeta.TypeVersion` is of type `FrameTypeVersion`, but there is no `FrameTypeVersion` property on the meta object).

Reply:
Got it, thank you!